src/misc/DefaultAvatar/DefaultAvatar.js

```diff
@@ -28,13 +28,18 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const DefaultAvatar = ({ userName, size }) => {
+export const DefaultAvatar = (props) => {
+  const { userName, size, variant } = props;
   const avatarBackgroundColor = stringToColor(userName);
   const letter = userName.charAt(0).toUpperCase();
 
-  const classes = useStyles({ avatarBackgroundColor, size });
+  const classes = useStyles({ avatarBackgroundColor, size, variant });
```

Comment:
We can not add variant in useStyles because it's not needed to class generation, we pass it directly to `variant` prop of `<Avatar />` component :slightly_smiling_face:

Reply:
Simple error...